io/pci/PowerVMEEH.py

```diff
@@ -59,12 +59,11 @@ def setUp(self):
         """
         if 'ppc' not in distro.detect().arch:
             self.cancel("Processor is not ppc64")
-        output = genio.read_file("/sys/kernel/debug/powerpc/eeh_enable")\
-            .strip()
-        if output != '0x1':
+        if 'PowerNV' in genio.read_file("/proc/cpuinfo").strip():
+            self.cancel("Test not supported on bare-metal")
+        if '0x1' not in genio.read_file("/sys/kernel/debug/powerpc/eeh_enable").strip():
             self.cancel("EEH is not enabled, please enable via FSP")
-            sys.exit(1)
-        self.max_freeze = int(self.params.get('max_freeze', default='1'))
+        self.max_freeze = self.params.get('max_freeze', default=1)
```

Comment:
This needs int() conversion since value is read from yaml file

Reply:
the parameter max_freeze value from yaml is of type int

scenario:
    max_freeze: 1

sample run
PARAMS (key=max_freeze, path=*, default=1) => 1
[stdout] <type 'int'>
[stdout]

Reply:
@abdhaleegit thanks, did not notice this.